src/plugins/condition/components/ConditionEdit.vue

```diff
@@ -1,18 +1,20 @@
 <template>
 <div v-if="condition"
-     class="c-cs-editui__conditions"
+     class="c-c-editui__conditions c-c-container__container c-c__drag-wrapper"
      :class="['widget-condition', { 'widget-condition--current': currentConditionIdentifier && (currentConditionIdentifier.key === conditionIdentifier.key) }]"
 >
     <div class="title-bar">
-        <span
-            class="c-c__menu-hamburger"
-            :class="{ 'is-enabled': !condition.isDefault }"
-            @click="expanded = !condition.expanded"
+        <span class="c-c__menu-hamburger"
+              :class="{ 'is-enabled': !condition.isDefault }"
+              :data-condition-index="conditionIndex"
```

Comment:
Is the `:data-condition-index` property needed? Could you use `this.conditionIndex` that has been passed into props directly in the dragstart method?

Reply:
done